### ISSUE TITLE
Dual extruder with different gear

### ISSUE THREAD

**Issue by paltoni** [MarlinFirmware/Marlin#4829]
Hi!
I have a printer with dual extruder but different gear and it not possible to change the step per unit separate for the E0 and the E1 in the firmware.Can you help solve this problem?

**Comment by Jinxycob** [comment]
ello,
  I've started messing around dual extruders, what  I discovered so far  is that, there only one Estep value for all extruders, so  possible solutions is to a)have identical extruders and b) use the flow modifier in your slicer to increase/decease flow values for the second extruder,,,
now am no expert so hoping am wrong  about the E steps!!!

**Comment by paltoni** [comment]
i found a firmware what can handle it, the name of this : Marlin kimbra

**Comment by thinkyhead** [comment]
Yes, we're well aware of MagoKimbra's extra steps-per-unit entries. We haven't taken that road, but may in the future.

**Comment by thinkyhead** [comment]
Yes, we know about MagoKimbra's extra steps-per-unit entries.
We haven't taken that road, but may in the future. It would be good to have this added.

The only solution in Marlin currently is to set a different flow multiplier for the second extruder. For example, if the steps per unit for E0 is 160 and the E1 needs 200, then you would use `M221 T1 S125` to set the flow rate to 125% for the second extruder. This affects all E movement, both forward and backward.

**Comment by thinkyhead** [comment]
#4849 coming soon…

**Comment by boelle** [comment]
since this is a duplicate why not close?

**Comment by thinkyhead** [comment]
Once #4849 is ready and gets merged it will be nice to have @paltoni give it a test run, so let's leave this open for now. It will be at least a few days before I have a chance to finish #4849 (which is incomplete), and with recent options for special types of extruders it's a little more complicated than it used to be, and will require being more meticulous.

**Comment by thinkyhead** [comment]
#5371 has been merged.
